src/sitemap.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset
+        xmlns="http://www.sitemaps.org/schemas/sitemap/0.9"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9
+            http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd">
+    <url>
+        <loc>http://bloomlibrary.org/</loc>
+        <changefreq>daily</changefreq>
+    </url>
+    <url>
+        <loc>http://bloomlibrary.org/#/landing</loc>
+        <changefreq>daily</changefreq>
+    </url>
+    <url>
+        <loc>http://bloomlibrary.org/#/support</loc>
+        <changefreq>daily</changefreq>
+    </url>
+    <url>
+        <loc>http://bloomlibrary.org/#/installers</loc>
+        <changefreq>daily</changefreq>
+    </url>
+    <url>
+        <loc>http://bloomlibrary.org/#/about</loc>
+        <changefreq>daily</changefreq>
+    </url>
+    <url>
+        <loc>http://bloomlibrary.org/#/opensource</loc>
+        <changefreq>daily</changefreq>
+    </url>
+    <url>
+        <loc>http://bloomlibrary.org/#/suggestions</loc>
+        <changefreq>daily</changefreq>
+    </url>
+    <url>
+        <loc>http://bloomlibrary.org/#/browse</loc>
+        <changefreq>daily</changefreq>
+    </url>
```

Comment:
Do we need to do anything to help it know to index the individual book descriptions/titles and that there are multiple pages of them?
I there anything we can do to help it know that the tags are important keywords for finding books?
Or that the languages list is important information for anyone searching for material in a particular language?
Not that any of this is a reason to delay merging what you've done...